pkg/rtc/mediatrackreceiver.go

```diff
@@ -34,7 +34,7 @@ import (
 	"github.com/livekit/livekit-server/pkg/rtc/types"
 	"github.com/livekit/livekit-server/pkg/sfu"
 	"github.com/livekit/livekit-server/pkg/sfu/buffer"
-	"github.com/livekit/livekit-server/pkg/sfu/dependencydescriptor"
+	"github.com/livekit/livekit-server/pkg/sfu/rtpextension/dependencydescriptor"
```

Comment:
most of the changes are grouping all extension stuff under `sfu/rtpextension` directory. So, a few things moved and that is most of the changes.